### ISSUE TITLE
Rancher UI: Show stack name in load balancer target

### ISSUE THREAD

**Issue by LukeHowellDev** [rancher/rancher#7973]
If you have multiple stacks that have similar service names it can be ambiguous as to which service is currently selected.  For example:

```
Stack: first-stack
  Service: app
Stack: second-stack
  Service: app
```

When either service is selected the select box would display `app`.  

I suggest displaying in the input box both the stack and the service name. Such as:

```
first-stack/app
```

**Comment by deniseschannon** [comment]
Available with `v1.6.15-rc6`

**Comment by tfiduccia** [comment]
Version - v1.6.15-rc6
Verified fixed
